src/routes/api/article/article.routes.js

```diff
@@ -22,7 +23,7 @@ const {
 router.post('/:articleId/favorite', [auth, confirmEmailAuth, validateId], FavoritesController.createOrRemoveFavorite);
 router.post('/', [auth, confirmEmailAuth], imageUpload.array('images', 10), validate(schema.articleSchema), articleController.createArticles);
 router.get('/', checkQuery, articleController.getAllArticles);
-router.get('/:slug', articleController.getOneArticle);
+router.get('/:slug', StatsWare.saveStat, articleController.getOneArticle);
```

Comment:
why did you change this? any user(logged in or not) can get a single article .

Reply:
That middle ware is transparent, doesn't return any responses but only scans for the auth header to ascertain loginstatus of requesting user. Couldn't put auth in that route because it is not supposed to be protected